client/app/views/steps/password.coffee

```diff
@@ -1,14 +1,24 @@
 StepView = require '../step'
+timezones = require '../../lib/timezones'
+
 
 module.exports = class PasswordView extends StepView
+
     template: require '../templates/view_steps_password'
 
     events:
-        'click button': 'onSubmit'
+        'click button': 'doSubmit'
```

Comment:
Why renaming here ? `onEvent` is a pretty much accepted naming for specifying event handlers (see https://www.reddit.com/r/javascript/comments/1uwjcp/naming_conventions_when_to_use_on_prefix/) and (http://stackoverflow.com/questions/11990176/backbone-js-best-practice-for-event-handlers-naming), but if you prefer to stick more and what has already been done in cozy-proxy, like here https://github.com/cozy/cozy-proxy/blob/development/client/app/views/register/import_google.coffee#L20, i suggest you to use `submit` instead. `doSubmit` is some kind of repetition :) (it is also what backbone documentation shows http://backbonejs.org/#View-events)

Thanks to fix this. :)